ext/src/content/content.js

```diff
@@ -268,50 +277,95 @@ function init() {
         }
 
         if (codeBlocks.length) {
-          /*
-           * TODO: Add support for language-all: <!-- language-all: lang-* -->
-           * Once this support is added, we can format inline code snippets (i.e. `const foo = 'bar';`).
-           * https://stackoverflow.com/editing-help#syntax-highlighting
-           */
+          let langAll = null;
+
+          // https://stackoverflow.com/editing-help#syntax-highlighting
           codeBlocks.forEach(lines => {
-            const codeBlockRegex = /^\s{0,3}(?:```|~~~)\s*lang-(.+)/u;
-            const indentedCodeRegex = /^\s*<!-- language: lang-(.+) -->/u;
+            const codeBlockRegex = /^\s{0,3}(?:```|~~~)\s*(?:lang-(.+))?/u;
+            const indentedCodeWithLangRegex = /^\s*<!-- language: lang-(.+) -->/u;
+            const langAllRegex = /^\s*<!-- language-all: lang-(.+) -->/u;
+            const codeSnippetRegex = /(`{1,2})([^\n`]+)(`{1,2})/gu;
             const [firstLine] = lines;
-            const [, lang = null] =
+
+            if (langAllRegex.test(firstLine)) {
+              [, langAll = null] = firstLine.match(langAllRegex);
+              return;
+            }
+
+            const [, lang = langAll] =
               firstLine.match(codeBlockRegex) ||
-              firstLine.match(indentedCodeRegex) ||
+              firstLine.match(indentedCodeWithLangRegex) ||
               [];
 
             if (!lang) {
               return;
             }
 
-            const isCodeBlock = codeBlockRegex.test(firstLine);
-            const indentedLineCodeBlockStartIdx = 2;
-            const codeLines = isCodeBlock
-              ? lines.slice(1, -1)
-              : lines.slice(indentedLineCodeBlockStartIdx);
-            let formattedBlock = window.prettier.format(codeLines.join("\n"), {
-              parser: PARSERS_LANG_MAP[lang],
-              plugins: window.prettierPlugins
-            });
-
-            // Prettier adds a trailing newline
-            if (codeLines.length !== formattedBlock.split("\n").length) {
-              formattedBlock = formattedBlock.replace(/\n$/u, "");
-            }
+            let formattedText = lines.join("\n");
+
+            // Code Snippets
+            if (codeSnippetRegex.test(firstLine)) {
+              formattedText = firstLine.replace(
+                codeSnippetRegex,
+                (match, openingBackticks, snippet, closingBackticks) => {
+                  let formattedSnippet = snippet;
+
+                  try {
```

Comment:
Wrapping this in a `try`/`catch` statement allows the plugin to continue onto the next code snippet or block without crashing. This will probably happen frequently since Prettier throws if the parser can't parse something.